test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -295,32 +521,132 @@ define([
-                        container.remove();
+                    itHasRowStructure();
+                });
+            });
+
+            // ensure the table structure is correct for a batch job with retries
+            describe('batch job with retries', () => {
+                const batchJob = JobsData.batchJob;
+                let allRows;
+
+                describe('table init', () => {
+                    beforeEach(async function () {
+                        container = document.createElement('div');
+                        this.jobManager = new JobManager({
+                            model: makeModel(JobsData.batchJob.jobArray),
+                            bus: Runtime.make().bus(),
+                        });
+                        this.jobStatusTableInstance = await createStartedInstance(container, {
+                            jobManager: this.jobManager,
+                        });
                     });
 
-                    itHasRowStructure();
+                    it('has the correct number of rows', () => {
+                        allRows = container.querySelectorAll('tbody tr');
+                        // data sanity check
+                        expect(Object.keys(batchJob.currentJobs).length).toEqual(
+                            Object.keys(batchJob.originalJobs).length
+                        );
+                        expect(Object.keys(batchJob.currentJobs).length).toEqual(allRows.length);
+                    });
+
+                    it('has the correct original rows', () => {
+                        allRows = Array.from(container.querySelectorAll('tbody tr'));
+                        // make sure all rows are present in batchJob.originalJobs
+                        const allIds = allRows.map((row) => {
+                            return row.id;
+                        });
+                        const expectedIds = Object.keys(batchJob.originalJobs).map((job_id) => {
+                            return `job_${job_id}`;
+                        });
+                        expect(allIds).toEqual(jasmine.arrayWithExactContents(expectedIds));
+                    });
+
+                    Object.keys(batchJob.currentJobs).forEach((job_id) => {
+                        it(`has the correct row content for job ${job_id}`, function () {
+                            this.input = batchJob.jobsById[job_id];
+                            this.job = this.input.retry_parent
+                                ? batchJob.jobsById[this.input.retry_parent]
+                                : batchJob.jobsById[job_id];
+                            this.row = container.querySelector(`#job_${this.job.job_id}`);
+                            // check the row content
+                            _checkRowStructure(this.row, this.job, this.input);
+                        });
+                    });
+                });
+                describe('table init with job info', () => {
```

Comment:
these tests ensure that the first column is populated correctly if the job params already exist.